osgtest/tests/test_380_cvmfs.py

```diff
@@ -11,7 +11,7 @@
 class TestCvmfs(osgunittest.OSGTestCase):
 
     __check_path = '/cvmfs/cms.cern.ch/cmsset_default.sh'
-    __cvmfs_image = '/cvmfs/singularity.opensciencegrid.org/opensciencegrid/osg-wn:3.3-el6'
+    __cvmfs_image = '/cvmfs/singularity.opensciencegrid.org/opensciencegrid/osgvo-el9:latest'
```

Comment:
what's this image used for? osgvo-el9 is 778 MB, about 3 times the size of osg-wn:3.3-el6 (221 MB).  If all we need is a runnable image then perhaps we could use library/alpine:latest

Reply:
Yeah, that's a good idea